scripts/merge-and-build.sh

```diff
@@ -141,6 +168,7 @@ echo "=========="
 tito tag --accept-auto-changelog
 export VERSION="v$(grep Version: origin.spec | awk '{print $2}')"
 echo ${VERSION}
+exit 0
 git push
 git push --tags
```

Comment:
We probably want this to be one command to make sure we can't crash between -- `git push --tags` should do both the objects and the refs